cmd/docs.go

```diff
@@ -37,18 +39,33 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.CliConfiguration{}, err)
 			}
 
+			// Detect terminal width if not specified in `atmos.yaml`
+			maxWidth := cliConfig.Settings.Docs.MaxWidth
+			if maxWidth == 0 {
+				if term.IsTerminal(int(os.Stdout.Fd())) {
+					w, _, err := term.GetSize(int(os.Stdout.Fd()))
+					if err == nil {
+						maxWidth = int(w)
+					}
+
+					if maxWidth > 120 {
+						maxWidth = 120
+					}
```

Comment:
The 120 max width seems arbitrary and we're not capping the width.

```suggestion
					if w > maxWidth {
						w = maxWidth
					}
```

Reply:
120 characters is a common standard width limit in terminal applications. It's wide enough to display most content comfortably while avoiding excessively long lines that become hard to read. This is how `glow` works as well

Reply:
I don't disagree it's a reasonable default, if no default is specified. As implemented, however, `maxWidth` is not the maxWidth, and there's an implicit max capped on the maxWidth, that seems awkward to me.

I think a conventional way of handling this would be:

1. If `maxWidth` is specified and greater than zero, we should set the width to the smaller (`min`) of screenWidth and maxWidth. Otherwise, if maxWidth is unset or zero, we should default to the `screenWidth`, capped at 120 as you have.
2. Remove max maxWidth to avoid adding an opinionated upper constraint over maxWidth, as this limits the flexibility of maxWidth to dynamically adjust based on the context.